cms/djangoapps/contentstore/management/commands/generate_test_course.py

```diff
@@ -0,0 +1,56 @@
+"""
+Django management command to generate a test course in a specific modulestore
+"""
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+from contentstore.management.commands.utils import user_from_str
+from contentstore.views.course import create_new_course_in_store
+from xmodule.modulestore import ModuleStoreEnum
+
+import json
+import random
+
+
+class Command(BaseCommand):
+    """ Generate a basic course """
+    help = 'Generate a course with settings on studio'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--json',
+            action='store',
+            dest='json',
+            type=str,
+            default=None,
+            help='A json object with the following fields: store, user, name, organization, number, run'
+        )
+
+    def handle(self, *args, **options):
+        # Set default course settings
+        store = "split"
+        user = user_from_str("edx@example.com")
+        name = "test-course"
+        org = "test-course-generator"
+        num = str(random.randint(0, 100))  # Random course number
```

Comment:
It would be. The course id is essentially a concatenation of the organization, number, and run, so as long as that's unique then if we don't specify settings, then we can still continue to run the command and not run into collision. But for default courses, the goal is still just to create a generic course, so I'll take suggestions on a better way to make a unique id